Automattic/wp-calypso
# Checkout: Remove WebPaymentBox from old checkout

#### Changes proposed in this Pull Request

Similar to https://github.com/Automattic/wp-calypso/pull/46876, this removes the `WebPaymentBox` (Apple Pay) from old checkout, since old checkout is no longer used. This is one step toward removing old checkout entirely.

#### Testing instructions

- Spoof the https request to localhost as wordpress.com (how to do this is beyond the scope of this guide).
- Visit checkout and choose Apple Pay.
- Verify that you get the Apple Pay dialog. (Testing completing the purchase is less important.)
